src/components/print/PrintDirective.js

```diff
@@ -951,6 +951,14 @@ goog.require('ga_time_service');
       }
     };
 
+    // Change a distance according to the change of DPI
+    var adjustDist = function(dist) {
+      if (!dist) {
+        return;
+      }
+      return dist * 90 / getDpi($scope.layout.name, $scope.dpi);
```

Comment:
90.7 is DPI in WMTS standard (used to calcualte scalefactor in WMTS GetCap).

Our WMTS Tiles are calculated with 96DPI and our WMS uses 96dpi as standard output. In print, we are using 150DPI.

Also, we use 72 DPI insided this print directive, I think to create the preview rectangle.

If I'm not completely mistaken, we should use 96 in this function here...or 72.